src/test/videopress/local-helpers/utils.js

```diff
@@ -123,47 +123,84 @@ export const expectShowMediaOptions = ( screen, { title, options } ) => {
 };
 
 /**
- * Generates the HTML of the VideoPress block.
+ * Generates the block settings related to privacy
+ * based on video and site privacy.
  *
  * @param {Object} options
- * @param {string} [options.guid]        VideoPress GUID.
- * @param {string} [options.title]       Title of the video.
- * @param {string} [options.description] Description of the video.
- * @param {string} [options.isPrivate]   True if the video is private.
+ * @param {string} [options.isVideoPrivate] True if the video is private. If not defined, it will use site's privacy.
+ * @param {string} [options.isSitePrivate]  True if the site is private.
+ */
+const generatePrivacySettings = ( {
+	isVideoPrivate,
+	isSitePrivate = false,
+} = {} ) => {
+	// If video privacy is not set, it will use the site privacy.
+	//
+	// Privacy setting options are:
+	// 0 = Public
+	// 1 = Private
+	// 2 = Site default
+	if ( typeof isVideoPrivate === 'undefined' ) {
+		return {
+			privacySetting: 2,
+			isPrivate: isSitePrivate,
+		};
+	}
+	return {
+		privacySetting: isPrivate ? 1 : 0,
```

Comment:
`isPrivate` appears to be undefined. Is this meant to be `isVideoPrivate` or `isSitePrivate` instead? This oversight may be causing (erroneous?) snapshot changes of `isPrivate` in this PR.

Reply:
Ah, my bad. This is meant to use `isVideoPrivate`. I'm afraid I renamed the parameter and forgot to update this part. I'll push a fix.

Reply:
I've just fixed this via https://github.com/wordpress-mobile/gutenberg-mobile/pull/5916/commits/2ab22a26e307bc47747083046d8faab230d5d81d.